itou/communications/dispatch/email.py

```diff
@@ -30,5 +21,26 @@ def build(self):
         )
 
     def send(self):
+        if (
+            # don't check if the user is still in the organization if it's already a forwarded notification
+            "forward_from_user" not in self.context
+            # Don't use should_send() if the user left the org because we don't want to use his settings
+            and self.is_applicable()
+            and self.user.is_prescriber
+            and self.structure
+        ):
+            memberships = (
+                PrescriberMembership.objects.active()
+                .filter(organization=self.structure)
+                .select_related("user")
+                .order_by("created_at")
```

Comment:
nit: J'imagine que l'`order_by` n'est là que pour les tests ?
`assert {mailoutbox[0].to, mailoutbox[1].to} == {admin_1.email, admin_2.email}`
ne me choquerait pas :)